src/core/sys/posix/aio.d

```diff
@@ -63,6 +63,19 @@ version (CRuntime_Glibc)
         }
     }
 }
+else version (OSX)
+{
+    struct aiocb
+    {
+        int aio_filedes;
+        off_t aio_offset;
+        void* aio_buf;
```

Comment:
The other branches have comments noting `aio_buf` is volatile. This is `volatile` on OSX too.